components/AssetFormTitle/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image';
+import {
+  TitleContainer,
+  Name,
+  General,
+  CollectionName,
+  CollectionCreator,
+  CollectionIconContainer,
+} from './AssetFormTitle.styled';
+
+type Props = {
+  name: string;
+};
+
+const AssetFormTitle = ({ name }: Props): JSX.Element => {
+  return (
+    <>
+      <CollectionIconContainer>
+        <Image
+          priority
+          layout="fixed"
+          width={24}
+          height={24}
+          src="/icon-monsters.png"
+          alt="Crypto Monsters icon"
+        />
+        <CollectionName>Crypto Monsters</CollectionName>
+      </CollectionIconContainer>
+      <TitleContainer>
```

Comment:
Can you double check the spacing here and the font colors?

Reply:
The spacing and font colors should show up on the Marketplace details page and My NFTs details page when reviewing WEB-833-my-template-detail. I'll leave this comment as unresolved to double check later when the other two PRs are reviewed/merged!